File: EadBackend/DTOs/ReviewDtos.cs
namespace EadBackend.DTOs
{
    public class CreateReviewDto
    {
        public string UserId { get; set; }
        public string ShopItemId { get; set; }
        public string ReviewText { get; set; }
        public int Stars { get; set; }
    }

    public class UpdateReviewDto
    {
        public string ReviewText { get; set; }
        public int Stars { get; set; }
    }

    public class ReviewDto
    {
        public string Id { get; set; }
        public string UserId { get; set; }
        public string ShopItemId { get; set; }
        public string ReviewText { get; set; }
        public int Stars { get; set; }
        public DateTime CreatedAt { get; set; }
    }
}

File: AndroidApp/app/src/main/java/com/example/teaapp/dto/ReviewDTOs.kt
package com.example.teaapp.dto
data class CreateReviewDTO(
    val userId: String,
    val shopItemId: String,
    val reviewText: String,
    val stars: Int
)

data class CreateReviewResponse(
    val id: String,
    val userId: String,
    val shopItemId: String,
    val reviewText: String,
    val stars: Int,
    val createdAt: String
)
